### PR TITLE
[docs] PR for tablet splitting: Add warning on manual splitting that it can lead to bad skew if misused 

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/preview/architecture/docdb-sharding/tablet-splitting.md [hunk 1]
```diff
@@ -167,79 +165,74 @@ Imagine there is a table with pre-existing data spread across a certain number o
...
+Automatic tablet splitting enables resharding of data in a cluster automatically while online, as well as transparently, when a specified size threshold has been reached.
 
-For details on the architecture design, see [Automatic Re-sharding of Data with Tablet Splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md).
+For details on the architecture design, see [Automatic re-sharding of data with tablet splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md).
 
 ### Enable automatic tablet splitting
 
-To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable_automatic_tablet_splitting) command, and specify the associated flags to configure when tablets should split.
+To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable_automatic_tablet_splitting) command and specify the associated flags to configure when tablets should split.
```
Comment (ddhodge) [inline docs/content/preview/architecture/docdb-sharding/tablet-splitting.md:213]: yb-master/#enable-automatic-tablet-splitting
also, on the yb-master page, these are referred to as flags

File: docs/content/preview/architecture/docdb-sharding/tablet-splitting.md [hunk 2]
```diff
@@ -167,79 +165,74 @@ Imagine there is a table with pre-existing data spread across a certain number o
...
 
-* In the **high phase**, each node has fewer than [`tablet_split_high_phase_shard_count_per_node`](../../../reference/configuration/yb-master/#tablet-split-high-phase-shard-count-per-node) shards. In this phase, YugabyteDB splits tablets larger than [`tablet_split_high_phase_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-high-phase-size-threshold-bytes).
+* In the high phase, each node has fewer than [`tablet_split_high_phase_shard_count_per_node`](../../../reference/configuration/yb-master/#tablet-split-high-phase-shard-count-per-node) shards. In this phase, YugabyteDB splits tablets larger than [`tablet_split_high_phase_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-high-phase-size-threshold-bytes).
 
 * Once the shard count exceeds the high phase count, YugabyteDB splits tablets larger than [`tablet_force_split_threshold_bytes`].
```
Comment (ddhodge) [inline docs/content/preview/architecture/docdb-sharding/tablet-splitting.md:221]: link?

**Review by ddhodge — APPROVED** [review]
LGTM couple comments

File: docs/content/preview/architecture/docdb-sharding/tablet-splitting.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/architecture/docdb-sharding/tablet-splitting.md:213]: Fixed

File: docs/content/preview/architecture/docdb-sharding/tablet-splitting.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/architecture/docdb-sharding/tablet-splitting.md:221]: Fixed

**Pull request merged**